Lib/platform.py

```diff
@@ -611,10 +611,13 @@ def _syscmd_file(target, default=''):
     try:
         output = subprocess.check_output(['file', target],
                                          stderr=subprocess.DEVNULL,
-                                         encoding='latin-1')
+                                         env={'LC_ALL': 'C'})
```

Comment:
Do you use LC_ALL=C to change the encoding? If yes, I'm not sure that it's a good idea.

If it's to get the output in english: I would be surprised since I'm not aware of a translated output of file?